### PR TITLE
Fix: Small screen cutoff text in Senior and Blind forms

### PR DESCRIPTION
**Pull request by krisrjohnson21**
We received feedback that the Senior and Blind Access forms were sometimes cut off on the right side of the screen for smaller mobile devices. This PR applies a CSS fix to reduce the width for ONLY screens between 250-375 pixels. 

Asana task: https://app.asana.com/0/1170867711449497/1202237097134092/f

### CODE REVIEW

**Review by arkadyan — COMMENTED** [review]
It looks like the built `css` files got checked in, but maybe not the source `scss` files?

File: build/stylesheets/Blind.css [hunk 1]
```diff
@@ -415,4 +415,11 @@ button.close:active:hover {
   color: #b3000f;
 }
 
+/* Reduce size for XS screens */
+@media screen and (min-width: 250px) and (max-width: 375px) {
```
Comment (arkadyan) [inline build/stylesheets/Blind.css:419]: Is the `min-width` even needed?

**Comment by krisrjohnson21** [comment]
> It looks like the built `css` files got checked in, but maybe not the source `scss` files?

Good catch! I'll update this now.

**Comment by arkadyan** [comment]
@krisrjohnson21 Have you tried running `npm run prettier`? I think you just have some whitespace noise.

File: cypress/automated-tests/youth-pass/address-section-required-fields.spec.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ describe('Youth Pass Address Section Required Fields', () => {
     // this test follows the by mail flow
     it('proceeds through an application', () => {
         const applicantZipCode = '02114';
-        const youthPassUrl = Cypress.env('youth_pass_url');
+        const youthPassUrl = 'https://mbta.prod.simpligov.com/prod/portal/ShowWorkFlow/AnonymousEmbed/e5e2dba6-5424-48d6-80d9-16a27e0fdd84';
```
[the remark this answers is not in the document]
Comment (krisrjohnson21) [reply cypress/automated-tests/youth-pass/address-section-required-fields.spec.js]: Nope, missed this one. Restored!

**Pull request merged**